elao.app.docker/.manala/github/system/action.yaml

```diff
@@ -86,6 +99,7 @@ runs:
             --file ./.manala/docker/compose/integration.yaml \
             --file ./.manala/docker/compose/cache.yaml \
             ${{ inputs.ssh_key != '' && '--file ./.manala/docker/compose/ssh-key.yaml' || '' }} \
+            ${{ inputs.env != '' && '--file ./.manala/docker/compose/env.yaml' || '' }} \
```

Comment:
Hm, using env file during container setup phase means that you'll never be able to use app/tier's related environment variables during exec phase. Is this really what you want ?

Reply:
That means you up a container for a specific app/tier, with this corresponding env vars and cannot be changed after setup, right.

I'm not sure what you mean by 

> you'll never be able to use app/tier's related environment variables during exec phase

Once the container is up with these env vars, those are available during exec phase.

Reply:
As double-checked, the container images are cached, but the env vars are properly injected at runtime from the given .env file 👌🏻